staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/fieldmanager_test.go

```diff
@@ -87,8 +78,12 @@ func TestApplyStripsFields(t *testing.T) {
 			"name": "b",
 			"namespace": "b",
 			"creationTimestamp": "2016-05-19T09:59:00Z",
+			"deletionTimestamp": "2016-05-19T09:59:00Z",
 			"selfLink": "b",
 			"uid": "b",
+			"clusterName": "b",
+			"generation": 0,
+			"managedFields": [],
```

Comment:
You want to see if `managedFields` gets removed "recursively", so you want to make this non-empty.